tests/testthat/test-use_cran_defaults.R

```diff
@@ -35,6 +35,6 @@ test_that("vendor_pkgs() vendors dependencies", {
 
   package_versions <- vendor_pkgs(path, quiet = TRUE)
   expect_snapshot(cat_file("src", "rust", "vendor-config.toml"))
-  expect_snapshot(package_versions)
```

Comment:
My last PR actually fixed versions here, can you roll this change back (and also snapshot)?